ml/src/test/scala/ai/h2o/sparkling/ml/models/MOJOParameterTestSuite.scala

```diff
@@ -102,6 +102,7 @@ class MOJOParameterTestSuite extends FunSuite with SharedH2OTestContext with Mat
       .setNumKnots(Array(5, 5))
       .setBs(Array(1, 1))
       .setScale(Array(.5, .5))
+      .setSplineOrders(Array(-1, -1))
```

Comment:
Just a question, are those values (-1, -1) meaningful ones?

Reply:
These values haven't been selected to improve performance algorithm, but are valid according to the tests on H2O-3 side:
https://github.com/h2oai/h2o-3/pull/6090/files#diff-4340a0b6fcce0ae9216a6d99f650d45af3f198370c43a422a9f7056707c104e1R233

Reply:
Well, I can't judge if combination of Bs, NumKnots and SplineOrders is a meaningful or not, but regarding the interface extension, this is OK (straightforward).